### PR TITLE
RHIDP-1478: Include HTTP Request as pre-loaded Plugin to RHDH

### PR DESCRIPTION
**Pull request by jesuino**
## Description

[RHIDP-1478](https://issues.redhat.com//browse/RHIDP-1478): Include HTTP Request as pre-loaded Plugin to RHDH

Adding a wrapper to the plugin roadiehq/scaffolder-backend-module-http-request.

## Which issue(s) does this PR fix

https://issues.redhat.com/projects/RHIDP/issues/RHIDP-1478

- Fixes [RHIDP-1478](https://issues.redhat.com//browse/RHIDP-1478)?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @roadiehq/scaffolder-backend-module-http-request --override-interop default"
+  },
+  "dependencies": {
+    "@backstage/backend-dynamic-feature-service": "^0.2.2",
```
Comment (davidfestal) [inline dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json:29]: I don't think you need this dependency.

File: dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "@backstage/backend-plugin-api": "0.6.13",
+    "@backstage/plugin-scaffolder-backend": "^1.22.3",
+    "@backstage/plugin-scaffolder-node": "^0.3.0",
+    "@roadiehq/scaffolder-backend-module-http-request": "4.1.10"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.25.2",
```
Comment (davidfestal) [inline dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json:36]: This should be the same dependency as for other wrappers

Comment (davidfestal) [reply dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json:36]: Same for the ` @janus-idp/cli` dependency.

File: dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @roadiehq/scaffolder-backend-module-http-request --override-interop default"
+  },
+  "dependencies": {
+    "@backstage/backend-dynamic-feature-service": "^0.2.2",
+    "@backstage/backend-plugin-api": "0.6.13",
```
Comment (davidfestal) [inline dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json:30]: Wouldn't it be possible to use on of the dependency versions already present in the `yarn.lock`: https://github.com/janus-idp/backstage-showcase/pull/1207/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR2481 ?

**Review by davidfestal — CHANGES_REQUESTED** [review]

**Comment by jesuino** [comment]
Hello @davidfestal  - thanks for your comments, I made the requested changes. Please let me know if I am missing something.

Thanks!

**Review by davidfestal — APPROVED** [review]

**Comment by davidfestal** [comment]
cc @gashcrumb

**Pull request merged**